setup.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+"""chaostoolkit builder and installer"""
+
+import sys
+import io
+
+import setuptools
+
+sys.path.insert(0, ".")
+from chaosext import __version__
+sys.path.remove(".")
+
+name = 'chaostoolkit-ext'
+desc = 'Chaos Toolkit Extension Template'
+
+with io.open('README.md', encoding='utf-8') as strm:
+    long_desc = strm.read()
+
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',  
+    'Intended Audience :: Developers',
+    'License :: Freely Distributable',
+    'Operating System :: OS Independent',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: Implementation',
+    'Programming Language :: Python :: Implementation :: CPython'
+]
+author = 'chaostoolkit Team'
+author_email = 'contact@chaostoolkit.org'
```

Comment:
might be empty string?

Reply:
No, I'd prefer this as a default for now. Obviously can be changed but just in case it isn't this can be a way we can offer support.

Reply:
With docs, this will be the next step.